webservices/common/views.py

```diff
@@ -25,12 +26,17 @@ class ApiResource(utils.Resource):
     join_columns = {}
     aliases = {}
     cap = 100
+    use_estimated_counts = True
 
     @use_kwargs(Ref('args'))
     @marshal_with(Ref('page_schema'))
     def get(self, *args, **kwargs):
         query = self.build_query(*args, **kwargs)
```

Comment:
It’s probably safer to fall back to estimates rather than exact. To avoid double negative, parameter name should be “use exact counts” defaulted to false or unset

Reply:
Double check more utils.resource endpoints

Reply:
Maybe consider putting the count logic all together in a function so we can test it more easily